infrastructure/cdn-in-a-box/traffic_vault/poststart.d/02-add-search-schema.sh

```diff
@@ -16,7 +16,7 @@
 # under the License.
 source /to-access.sh
 
-curl -kvs -XPUT -H 'Content-Type:application/xml' "https://$TV_ADMIN_USER:$TV_ADMIN_PASSWORD@$TV_FQDN:$TV_HTTPS_PORT/search/schema/sslkeys" -d @/sslkeys.xml 
+curl -kvs -XPUT -H 'Content-Type:application/xml' "https://$TV_ADMIN_USER:$TV_ADMIN_PASSWORD@$TV_FQDN:$TV_HTTPS_PORT/search/schema/sslkeys" -d @/sslkeys.xml || cat /var/log/riak/*
```

Comment:
I am not sure I understand why are we choosing data set as ssl keys or riak log? shouldn't it be only ssl keys unless riak logs = ssl keys

Reply:
The or here is not being applied to what data to use, it's the second-lowest shell operator binding. So this isn't
```bash
curl -kvs -XPUT -H 'Content-Type:application/xml' "https://$TV_ADMIN_USER:$TV_ADMIN_PASSWORD@$TV_FQDN:$TV_HTTPS_PORT/search/schema/sslkeys" -d (@/sslkeys.xml || cat /var/log/riak/*)
```
(which I don't think would even actually work like that), it's
```bash
(curl -kvs -XPUT -H 'Content-Type:application/xml' "https://$TV_ADMIN_USER:$TV_ADMIN_PASSWORD@$TV_FQDN:$TV_HTTPS_PORT/search/schema/sslkeys" -d @/sslkeys.xml) || (cat /var/log/riak/*)
```
that is, the second command runs if the first one fails.

Reply:
oh ok.